iphone/Classes/AnalyticsModule.m

```diff
@@ -105,6 +105,17 @@ - (void)filterEvents:(id)args
   }
 }
 
+- (void)setOptedOut:(id)optedOut
+{
+  ENSURE_TYPE(optedOut, NSNumber);
+  [[APSAnalytics sharedInstance] setOptedOut:[TiUtils boolValue:optedOut]];
+}
+
+- (void)optedOut
+{
+  @([[APSAnalytics sharedInstance] isOptedOut]);
```

Comment:
1. Isn't it good if we make this method's return type to (NSNumber *) for more readability and return it.
2. Following test case crashing, if I close the app and run again. Can this not be a test case ?
`var win = Ti.UI.createWindow({
    backgroundColor: '#fff'
});

var btn = Ti.UI.createButton({
    title: 'Trigger'
});

btn.addEventListener('click', function() {
	Ti.Analytics.setOptedOut(true);
    });

win.add(btn);
win.open();
`